samples/nrf9160/sms/prj.conf

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# General config
+
+CONFIG_SMS_SEND_PHONE_NUMBER=""
```

Comment:
`""` is the default, it's redundant to specify here.

Reply:
True, I added it here more for documentation purposes so a user can easily find what (s)he needs to change in order to see a sent SMS message from the sample.
I can definitely remove this if preferred but I'd be more than happy to have it readily available here if I was a user of the sample.